src/G2/Equiv/EquivADT.hs

```diff
@@ -36,6 +36,20 @@ proofObligations :: HS.HashSet Name ->
 proofObligations ns s1 s2 e1 e2 =
   exprPairing ns s1 s2 e1 e2 HS.empty [] []
 
+removeTicks :: Expr -> Expr
+removeTicks (Tick _ e) = removeTicks e
+removeTicks e = e
+
+removeAllTicks :: Expr -> Expr
+removeAllTicks = modifyASTs removeTicks
+
+unAppNoTicks :: Expr -> [Expr]
+unAppNoTicks e =
+  let e_list = unApp e
+  in case e_list of
+    e':t -> (removeTicks e'):t
+    _ -> e_list
```

Comment:
Do we still need these functions now that we add the stack tick immediately before symbolic execution, rather than immediately after?

Reply:
I expect that we may still need these functions since we still have recursion ticks and the ticks for identifying case statements.

Reply:
Hmm, maybe... aren't those recursion ticks always in the same places in the Expr's though?